app/modes/initmodes.F90

```diff
@@ -181,6 +183,17 @@ subroutine initProgramVariables()
     ! oscillation cycles in an animation
     nCycles = 3
 
+    ! Eigensolver
+    call getChildValue(root, "EigenSolver", buffer2, "qr")
+    select case(tolower(char(buffer2)))
+    case ("qr")
+      iSolver = 1
```

Comment:
Can we use the `electronicSolverTypes` enumerator (as in the main code) here?

Reply:
Not sure, as there are various density matrix/transport solvers that don't make much sense. Will think about it.

Reply:
done